packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelog.js

```diff
@@ -66,10 +66,11 @@ module.exports = function generateChangelog( newVersion = null ) {
 					merges: undefined,
 					firstParent: true
 				};
-				const parserOpts = require( '../changelog/parser-options' );
-				const writerOpts = require( '../changelog/writer-options' );
+				const parserOptions = require( '../utils/parser-options' );
+				const writerOptions = require( '../utils/writer-options' );
+				writerOptions.transform = require( '../utils/transformcommitforckeditor5package' );
```

Comment:
You must not touch an object which was returned to you by reference. This way you're affecting someone else's `writerOptions` too. So either make `parser-options` and `writer-options` functions which return a new instance every time or clone the object here.